package.json

```diff
@@ -64,7 +64,7 @@
     "check-types": "npm-run-all check-types:*",
     "test": "npm-run-all test:*",
     "build": "npm-run-all build:*",
-    "dev": "cross-env NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6006",
+    "dev": "cross-env NODE_OPTIONS=--openssl-legacy-provider storybook dev -p 6006",
```

Comment:
**question**: can you check if we still need `NODE_OPTIONS=--openssl-legacy-provider` ?

Reply:
hmm it seems to be working but I would recommend to change it afterwards and double check that. Bumbing storybook is big enough for one PR